python_modules/dagit/dagit/graphql.py

```diff
@@ -136,6 +137,7 @@ async def graphql_ws_endpoint(self, websocket: WebSocket):
         """
         observables = {}
         tasks = {}
+        event_loop = get_event_loop()
```

Comment:
just noting that this is pseudo-deprecated https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_event_loop

using get_running_loop and explicitly setting up the loop may be better?